### DISCUSSION TITLE
微调结果的部署为什么最新代码没有了？

### DISCUSSION THREAD

**Discussion by shenlongju11** [zai-org/ChatGLM3#1140] [Q&A] [answered]
看到24年1月份的帖子，还在介绍微调结果的部署到大模型，从截图可以看出有相关的demo代码。但是最新的代码为什么删除了部署的介绍？哪里可以学到如何部署？

**Comment by zRzRzRzRzRzRzR** [comment]
有啊, 代码也都还在finetune_demo 内

**Reply by chenchen333-dev** [reply]
为什么推理微调后的模型 它不会微调时用过的训练数据，就好像没微调一样
![image](https://github.com/THUDM/ChatGLM3/assets/73687307/5294977c-dfc2-4b15-b0e1-67815559804c)
原数据集回答为1354.51

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
300而已，基本没学吧
